nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -132,46 +131,46 @@ def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
 
     @staticmethod
     @abstractmethod
-    def get_bias_value(model: TModel, node: NNCFNode) -> np.ndarray:
+    def get_bias_value(model: TModel, bias_node: NNCFNode) -> np.ndarray:
         """
         Returns bias value in the NumPy format of provided node.
 
         :param model: Backend-specific model for the initializer finding.
-        :param node: Node of NNCFGraph with bias value.
+        :param bias_node: Node of NNCFGraph with bias value.
         :return: Bias value in the NumPy format.
         """
 
     @staticmethod
     @abstractmethod
-    def get_bias_port_id(model: TModel, node: NNCFNode) -> int:
+    def get_bias_port_id(model: TModel, bias_node: NNCFNode) -> int:
         """
         Returns bias Port ID corresponding to the node.
 
         :param model: Backend-specific model.
-        :param node: Node of NNCFGraph with bias value.
+        :param bias_node: Node of NNCFGraph with bias value.
         :return: Port ID corresponding to bias.
         """
 
     @staticmethod
     @abstractmethod
-    def get_activation_port_ids_for_bias_node(model: TModel, node: NNCFNode) -> Tuple[int, int]:
+    def get_activation_port_ids_for_bias_node(model: TModel, biased_node: NNCFNode) -> Tuple[int, int]:
```

Comment:
Please, be a consistency. If you use the `bias_node` everywhere then you should use it here as well.

Reply:
This is the point. `bias_node` and `biased_node` is not the same. 
- `bias_node` - this is the node with the bias value. For example, for the OV backend, it's the `Add` after `Conv`, for the ONNX it's the `Conv` itself.
- `biased_node` - this is the node that would be affected by bias. For OV it's the `Conv`, as for the ONNX.

Reply:
It's become a bit clearer, but I find the following in the docstrings:
```
;param biased_node: Node of NNCFGraph with bias value.
:param bias_node: Node of NNCFGraph with bias value.
```

Reply:
I see. My bad. Fixed.